### PR TITLE
feat: Publishing Profiles for LinkedIn

### PR DESCRIPTION
**Pull request by jnv**

### CODE REVIEW

File: src/sf/use-cases.js [hunk 1]
```diff
@@ -10,6 +15,7 @@ function checkAllowedProvider(provider) {
 
 async function getProfilesForPublishing(providerName, accessToken) {
   checkAllowedProvider(providerName);
+  console.log('getProfilesForPublishing', providerName, accessToken)
```
Comment (janhalama) [inline src/sf/use-cases.js]: debug log should be removed

File: src/sf/providers/linkedin.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (janhalama) [inline src/sf/providers/linkedin.json:1]: I miss Linkedin setup documentation in README.md

File: src/sf/grid/social-media/publishing-profiles.linkedin.suma [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        detail = body.message
+      }
+    }
+  }
+}
+
+operation MapOrganization {
+  org = args.element['organization~']
+  orgLogo = null
+  // FIXME: What if it's empty?
```
Comment (janhalama) [inline src/sf/grid/social-media/publishing-profiles.linkedin.suma]: This is what happens with my profile which has not logo set. Following set if condition should fix it.

```
set if (org.logoV2 && org.logoV2['cropped~']) {
    logoElements = org.logoV2['cropped~'].elements
}
```

**Review by janhalama — COMMENTED** [review]
Looks good 👍. I have just a few comments.

File: src/sf/grid/social-media/publishing-profiles.linkedin.suma (hunk 3 above)
Comment (jnv) [reply src/sf/grid/social-media/publishing-profiles.linkedin.suma]: Looks like if the profile has not image, then `logoV2` will be completely missing. Surprisingly this didn't cause a map crash, but I will put a guard there anyway.

File: src/sf/providers/linkedin.json (hunk 2 above)
Comment (jnv) [reply src/sf/providers/linkedin.json:1]: I would like to address that in a separate PR.

**Review by janhalama — APPROVED** [review]

**Pull request merged**